client/devdocs/design/search-collection.jsx

```diff
@@ -102,7 +106,12 @@ const Collection = ( {
 
 		return (
 			<div>
-				<DocsExampleWrapper name={ exampleName } unique={ !! component } url={ exampleLink }>
+				<DocsExampleWrapper
+					name={ exampleName }
+					unique={ !! component }
+					url={ exampleLink }
+					pageScroll={ scroll }
```

Comment:
I think the prop's name should be more descriptive of how it applies to the component, not what the passed method does. Can you name the prop `onTitleClick`